lib/components/index.ts

```diff
@@ -49,6 +49,7 @@ export { Strong } from './Strong/Strong';
 export { Tag } from './Tag/Tag';
 export { Text } from './Text/Text';
 export { Textarea } from './Textarea/Textarea';
+export { TextDropdown } from './TextDropdown/TextDropdown';
 export { TextField } from './TextField/TextField';
```

Comment:
Interesting side note—I feel like the name `TextField` is now feeling out of step because it has nothing to do with the `Text` component.